src/main/java/org/alliancegenome/curation_api/model/entities/DiseaseAnnotation.java

```diff
@@ -36,7 +59,6 @@
 @Entity
 @Data
 @EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
-//@ToString(exclude = {"genomicLocations"})
```

Comment:
Was this intentional?

Reply:
It was commented out so I just removed it.

Reply:
Ahh ok... I think we should bring it back the idea is that every class should have it. Might need to adjust the exclude.

Reply:
It's back now